packages/javascript/bh-shared-ui/src/components/HelpTexts/ADCSESC3/General.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { FC } from 'react';
+import { groupSpecialFormat } from '../utils';
+import { EdgeInfoProps } from '../index';
+import { Typography } from '@mui/material';
+
+const General: FC<EdgeInfoProps> = ({ sourceName, sourceType, targetName }) => {
+    return (
+        <>
+            <Typography variant='body2'>
+                {groupSpecialFormat(sourceType, sourceName)} the capability to perform the ADCS ESC3 attack against the
+                domain {targetName}.
+            </Typography>
+            <Typography variant='body2'>
+                The principal has permission to enroll on a certificate template with the Certificate Request Agent EKU
+                (OID 1.3.6.1.4.1.311.20.2.1), allowing them to obtain an enrollement agent certificate. They also have
```

Comment:
looks like there is an extra e in enrollment... ?

Reply:
Good catch! I've removed the extra e